### ISSUE TITLE
Gepard SPAAG Corrections

### ISSUE THREAD

**Issue by FrangibleCover** [PygmalionOfCyprus/cmo-db-requests#2652]
### DB Selector

Both

### Affected DBID(s)

All Gepard variants

### Summary of Changes

Gepards actually come in a lot of variants, as follows:

Gepard
- Exactly as it is in the current DB
- Germany, Belgium (as 'Gepard B2')

Gepard A1
- Add a Laser Rangefinder to each mount and make whatever accuracy adjustments are appropriate
- Germany, Belgium (as 'Gepard B2LV')
- In service in parallel to base Gepard (also for CWDB therefore)

Gepard A2
- Significant overhaul of A1. Adds better FCS, improved C3 (better reaction time), thermal sight.
- Germany, Brazil, Ukraine (2022), probably Romania, probably Chile (2008-2011)
- Entered service 1998.

PRTL 35mm
- As Gepard, with Hollandse Signaalapparaten radars: Search radar is rated 15km I-Band, Track radar is rated 13km, I/J band. Allegedly better than the German ones.
- Netherlands
- Comes in a platoon of 3x vehicles
![image](https://user-images.githubusercontent.com/109925704/206869715-e899498c-3467-4670-ae9d-b0d2f2efd0c6.png)

PRTL 35mm GWI 'Cheetah'
- As PRTL, with the upgrades of Gepard A2. Joint project ensured coherence.
- Netherlands, Jordan

It's nice to do something with decent sourcing for a change.

### Sources

https://www.army-technology.com/projects/gepard/
https://weaponsystems.net/system/69-Gepard
https://en.wikipedia.org/wiki/Flakpanzer_Gepard
https://www.inetres.com/gp/military/cv/ada/Gepard.html
https://www.armyrecognition.com/december_2012_new_army_military_defence_industry/brazilian_army_will_acquire_36_gepard_german-made_35mm_anti-aircraft_armoured_vehicles_1012122.html
https://www.joint-forces.com/world-news/defence-news/53405-gepard-self-propelled-anti-aircraft-guns-for-ukraine
http://www.army-guide.com/eng/product229.html
https://nl.wikipedia.org/wiki/Cheetah_PRTL
https://de.wikipedia.org/wiki/Flugabwehrkanonenpanzer_Gepard

**Comment by KLABDB3K** [comment]
Waiting for Sgt York DIVADS v2.0 to be reincarnated next based on operational effectiveness and cost effectiveness of Gepard against UCAV/UAV.

**Comment by FrangibleCover** [comment]
Yeah, Gepard A2 is doing well out there. I wonder if basic 80s Gepard would produce the same results, it's cheaper but it's noticeably worse.

**Comment by PygmalionOfCyprus** [comment]
Handled 502.
